test/src/main/java/org/apache/accumulo/test/ThriftServerBindsBeforeZooKeeperLockIT.java

```diff
@@ -94,7 +94,7 @@ public void testMonitorService() throws Exception {
             final int responseCode = cnxn.getResponseCode();
             final String errorText = FunctionalTestUtils.readAll(cnxn.getErrorStream());
             // This is our "assertion", but we want to re-check it if it's not what we expect
-            if (HttpURLConnection.HTTP_UNAVAILABLE == responseCode && null != errorText && errorText.contains(BasicServlet.STANDBY_MONITOR_MESSAGE)) {
+            if (HttpURLConnection.HTTP_UNAVAILABLE == responseCode && null != errorText && errorText.contains("This is not the active Monitor")) {
```

Comment:
Honestly, I don't expect this test to pass anymore. I inline'd the variable so I could delete its source class, before I offered that change as a PR to @lstav 's branch. I need to revisit this and figure out what the test was checking for, and see whether the checks still make sense for the current behavior of the monitor.

After this PR, there shouldn't be any reason the monitor would be unavailable. However, as a secondary monitor, it won't be receiving logs, and we should probably have a message on the log page for that, at the very least (maybe also a special symbol in the menu to indicate that it's not receiving logs at the moment).